R/expectation.R

```diff
@@ -86,12 +90,25 @@ format.expectation_success <- function(x, ...) {
 
 #' @export
 format.expectation_error <- function(x, ...) {
-  paste(c(x$message, create_traceback(x$call)), collapse = "\n")
+  format_with_trace(x)
```

Comment:
Can `create_traceback()` now be deleted?

Reply:
There's still some work on that front because the current mechanism of capturing calls is tied to a reporter that gets you into `recover()`.

Reply:
Now removed, as well as `frame_calls()`.